geomesa-accumulo/geomesa-accumulo-distributed-runtime/pom.xml

```diff
@@ -28,24 +28,6 @@
             <artifactId>geomesa-accumulo-datastore_2.11</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-accumulo-raster_2.11</artifactId>
```

Comment:
@elahrvivaz it was partly my suggestion here.  It seemed like having a Vector and a Vector+Raster jar would be the easiest thing to do.  Worst case, we can leave this until later.